.github/workflows/release_trigger.yml

```diff
@@ -13,15 +13,15 @@ jobs:
       - name: Setup Environment
         id: setup
         uses: actions/github-script@v6
-        with: 
-          script: | 
+        with:
+          script: |
             if ('${{ github.event.action }}' === 'prereleased') {
               core.setOutput('environment', 'staging')
             }
             if ('${{ github.event.action }}' === 'released') {
               core.setOutput('environment', 'prod')
             }
```

Comment:
I committed this without thinking about it, but I like the `===` in here for JS comparisons vs the `==` that was suggested. 

```=== (Triple equals) is a strict equality comparison operator in JavaScript, which returns false for the values which are not of a similar type. This operator performs type casting for equality. If we compare 2 with “2” using ===, then it will return a false value.```

This just adds that little bit of extra validation to the data type check and I wanted that in here.